src/ast/scopes/Scope.ts

```diff
@@ -2,20 +2,14 @@ import { AstContext } from '../../Module';
 import Identifier from '../nodes/Identifier';
 import { ExpressionEntity } from '../nodes/shared/Expression';
 import { UNDEFINED_EXPRESSION } from '../values';
-import ArgumentsVariable from '../variables/ArgumentsVariable';
-import ExportDefaultVariable from '../variables/ExportDefaultVariable';
 import LocalVariable from '../variables/LocalVariable';
-import ThisVariable from '../variables/ThisVariable';
 import Variable from '../variables/Variable';
 import ChildScope from './ChildScope';
 
 export default class Scope {
 	children: ChildScope[] = [];
 	variables: {
 		[name: string]: Variable;
-		arguments?: ArgumentsVariable;
```

Comment:
Type change. The compiler complained that these properties had a different type than the index signature. The alternative would've been to make the index signature `[name: string]: Variable | undefined;`, which introduced errors elsewhere.

This change didn't introduce any new  type errors so should be okay?